ledger/participant-integration-api/src/main/scala/platform/store/backend/common/CommonStorageBackend.scala

```diff
@@ -205,15 +204,32 @@ private[backend] trait CommonStorageBackend[DB_BATCH] extends StorageBackend[DB_
                                                      |update parameters set participant_pruned_up_to_inclusive={pruned_up_to_inclusive}
                                                      |where participant_pruned_up_to_inclusive < {pruned_up_to_inclusive} or participant_pruned_up_to_inclusive is null
                                                      |""".stripMargin)
+  private val SQL_UPDATE_MOST_RECENT_PRUNING_INCLUDING_ALL_DIVULGED = SQL("""
```

Comment:
Note: there are three different ways how the same concept is named in the code:
1. `participant_all_divulged_contracts_pruned_up_to_inclusive`
2. `updatePrunedAllDivulgenceEventsUpToInclusive`
3. `SQL_UPDATE_MOST_RECENT_PRUNING_INCLUDING_ALL_DIVULGED`

Consider improving readability by deciding on one way to refer to the field, and use that name consistently.